### PR TITLE
Updated README.md, adds Authors, renames Contributors

### PR DESCRIPTION
**Pull request by JamesKunstle**
This PR updates our README so that newcomers have a more digestible first impression.

Also names major contributors. Would love to know who else to include, I'm not sure who wants to be included publicly.

Signed-off-by: James Kunstle <jkunstle@bu.edu>

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Current Team
+
+* **Cali Dolfi** : cdolfi@redhat.com
+* **James Kunstle** : jkunstle@redhat.com
+* **Sean Goggins**
```
Comment (cdolfi) [inline AUTHORS.md:5]: I would ask each person for consent

File: README.md [hunk 2]
```diff
@@ -3,66 +3,105 @@
...
 
-*tl;dr sandiego-rh/explorer organizes the visualizations and data generated in the sandiego-rh/sandiego repo as a web-app.*
+## Introduction
 
-[Example of Application](https://eightknot.osci.io/)
+8knot is a [Dash](https://dash.plotly.com/) data web-app built by Red Hat's Open Source Program Office ([OSPO](https://www.redhat.com/en/blog/channel/red-hat-open-source-program-office)).
 
-The number of repositories and projects that GitHub hosts is staggering. Even more impressive is the rate at which this number is growing.
-Projects on GitHub range some student projects that are likely to only be updated a handful of times ever to technological monoliths like the
-Linux Kernel. Understanding the growth trajectory, contributor behavior, progress blockers, etc. intuitively is no longer feasible for individual
```
Comment (cdolfi) [inline README.md:15]: Reason for taking this entirely out?

File: README.md [hunk 3]
```diff
@@ -3,66 +3,105 @@
...
+All runtime environment installation and architecture is handled in Docker containers.
+
+Anything not handled by Docker itself (building/networking/parameterizing containers) has
+been handled by the script "scripts/launch\_dev.sh". Please take a moment to become familiar with the steps in this script and suggest any improvements.
+
+Please run the following to launch your app:
 
 ```bash
-bash scripts/run_docker_container.sh
+bash scripts/launch_dev.sh
```
Comment (cdolfi) [inline README.md:101]: Is this now functioning for the updated architecture?

**Review by cdolfi — CHANGES_REQUESTED** [review]

File: AUTHORS.md (hunk 1 above)
Comment (JamesKunstle) [reply AUTHORS.md:5]: 👍 okay good thought

File: README.md (hunk 2 above)
Comment (JamesKunstle) [reply README.md:15]: do you like it? I removed it because it was verbose and I didn't like my own writing there but I would keep it in if you like it

File: README.md (hunk 3 above)
Comment (JamesKunstle) [reply README.md:101]: yep, sean merged the working from in the last PR I made

File: README.md (hunk 2 above)
Comment (cdolfi) [reply README.md:15]: Personally I liked it. If you want to make some edits to fit better for you I would. I think the additional context is useful

Comment (JamesKunstle) [reply README.md:15]: Okay I kept it in, that's good feedback. I made a little edit but it's intact.

**Review by cdolfi — APPROVED** [review]

**Comment by cdolfi** [comment]
All this need is the addition context from our meeting today and itll be good to go

File: AUTHORS.md (hunk 1 above)
Comment (JamesKunstle) [reply AUTHORS.md]: done
